test/inductor/test_torchinductor.py

```diff
@@ -800,6 +800,76 @@ def repeat(x, n):
         self.assertEqual(expect, actual)
         self.assertEqual(actual, repeat(x, 3))
 
+    @skipIfRocm
+    def test_neg_index(self):
+        def test(fn, inps, has_assert: bool, has_wrapping=True):
+            for dynamic in (True, False):
+                fn_opt = torch.compile(dynamic=dynamic)(fn)
+                if self.device == "cpu":
+                    code = run_and_get_cpp_code(fn_opt, *inps)
+                    lines = code.split("\n")
+                    found = False
+                    if "Vectorize" not in code:
+                        for line in lines:
+                            # match ternary operator
+                            if "?" in line:
+                                pos = line.find("?")
+                                found |= ":" in line[pos + 1 :]
```

Comment:
Just regex this

Reply:
Also, are there any other ?'s in the code, or could you simply try to find a ? in the code?